go.mod

```diff
@@ -261,6 +259,10 @@ require (
 	github.com/multiformats/go-varint v0.0.7 // indirect
 	github.com/nxadm/tail v1.4.8 // indirect
 	github.com/olekukonko/tablewriter v0.0.5 // indirect
+	github.com/onflow/flow-ft/lib/go/contracts v0.7.1-0.20240213220156-959b70719876 // indirect
+	github.com/onflow/flow-ft/lib/go/templates v0.7.1-0.20240213220156-959b70719876 // indirect
+	github.com/onflow/flow-nft/lib/go/contracts v1.1.1-0.20240214230837-cd2c42e54b4a // indirect
+	github.com/onflow/flow-nft/lib/go/templates v0.0.0-20240213205729-48f42d9896f8 // indirect
```

Comment:
Should the versions here be the same?

Reply:
I actually removed all the usage of the flow-nft and flow-ft repos, so the only places I am seeing it is in the go.mod and go.sum files. That means we can just remove these dependencies from all the go.mod files completely, right? I just put getters for all of these contracts and transactions in the core-contracts package so we were getting them all from the same place

Reply:
thats just because some commits updated transactions and some updated contracts. It is using all of the correct ones though, so not a problem. I can make them all match if that would make you feel better though

Reply:
okay. makes sense. I updated everything locally and synced with `feature/stable-cadence` and I'm running the tests locally now